doc/man/pkgdev

```diff
@@ -1 +1 @@
-/home/arthur/src/pkgcore/pkgdev/doc/generated/pkgdev
\ No newline at end of file
+/home/ferringb/gentoo/pkgdev/doc/generated/pkgdev
```

Comment:
that should not be in VCS.

Reply:
I'm not entirely sure why git is picking up the content through the symlink; more to the point, why is there a symlink here?  @arthurzam can you give some clarity on what you were working around so I can take a stab at tweaking sphinx rather than symlinking?

Reply:
that looks like a mistaknly added file to repo. Should be dropped.

Reply:
Wiped it.  We probably want a .gitignore for this, although reworking the sphinx man page extension is probably wiser.

If you check the other projects, you'll find symlinks in there too- all pointing back to the built. content.